docs/usage/control_plane_migration.md

```diff
@@ -21,4 +20,6 @@ If the `Shoot` has different `.spec.seedName` and `.status.seedName` a process i
 2. Kubernetes API Server is stopped and the extension resources are annotated with `gardener.cloud/operation=migrate`.
 3. Full snapshot of the ETCD is created and terminating of the Control Plane in the `Source Seed` is initiated.
 
-If the process is successful, we update the status of the `Shoot` by setting the `.status.seedName` to the null value. That way, a restoration is triggered in the `Destination Seed` and `.status.lastOperation` is changed to `Restore`.
\ No newline at end of file
+If the process is successful, we update the status of the `Shoot` by setting the `.status.seedName` to the null value. That way, a restoration is triggered in the `Destination Seed` and `.status.lastOperation` is changed to `Restore`. The control plane migration is completed when the `Restore` operation has completed successfully.
+
+By default the shoot's etcd backups will continue to be uploaded to the `BackupBucket` of the `Source Seed` after control plane migration has finished. If you want the etcd backups to instead be uploaded to the `BackupBucket` of the `Destination Seed`, you have to enable the `CopyEtcdBackupsDuringControlPlaneMigration` feature gate on the `gardenlet` by adding the following command flag: `--feature-gates=CopyEtcdBackupsDuringControlPlaneMigration=true`. Note that this will also move the existing etcd backups to the `BackupBucket` of the `Destination Seed` during the control plane migration porcess.
```

Comment:
```suggestion
By default the shoot's etcd backups will continue to be uploaded to the `BackupBucket` of the `Source Seed` after control plane migration has finished. If you want the etcd backups to instead be uploaded to the `BackupBucket` of the `Destination Seed`, you have to enable the `CopyEtcdBackupsDuringControlPlaneMigration` feature gate on the `gardenlet` by adding the following command flag: `--feature-gates=CopyEtcdBackupsDuringControlPlaneMigration=true`. Note that this will also move the existing etcd backups to the `BackupBucket` of the `Destination Seed` during the control plane migration process.
```